vitest.config.ts

```diff
@@ -12,7 +12,7 @@ export default defineConfig({
     tsconfigPaths(),
   ],
   test: {
-    include: ['src/**/*.spec.{js,jsx,ts,tsx}'],
+    include: ['src/screens/UserPortal/Volunteer/Actions/Actions.test.tsx'],
```

Comment:
Please revert this change. We need to be testing all file types. This will cause long term issues with our code base

Reply:
ok